gwt-client-common/src/main/java/nl/aerius/wui/history/HistoryManager.java

```diff
@@ -22,7 +22,24 @@
 
 @ImplementedBy(HistoryManagerImpl.class)
 public interface HistoryManager {
+
+  /**
+   * Reads the current place, and runs goto that place.
```

Comment:
Not sure about this description. 'Determines current place based on history and goes to that place' might make more sense?